packages/driver/src/cypress/error_messages.ts

```diff
@@ -1710,6 +1710,31 @@ export default {
 
       This is likely because the data argument specified is not serializable. Note that functions and DOM objects cannot be serialized.`,
     },
+    // TODO: These deprecation warnings and forbidden use errors need to be audited before releasing multi-domain
+    route: {
+      forbidden: {
+        message: `${cmd('route')} has been deprecated and use is forbidden in ${cmd('switchToDomain')}. Consider migrating to using ${cmd('intercept')} instead.`,
+        docsUrl: 'https://on.cypress.io/intercept',
+      },
+    },
+    server: {
+      forbidden: {
+        message: `${cmd('server')} has been deprecated and use is forbidden in ${cmd('switchToDomain')}. Consider migrating to using ${cmd('intercept')} instead.`,
+        docsUrl: 'https://on.cypress.io/intercept',
+      },
+    },
+    Server: {
+      forbidden: {
+        message: `\`Cypress.Server.*\` has been deprecated and use is forbidden in ${cmd('switchToDomain')}. Consider migrating to using ${cmd('intercept')} instead.`,
+        docsUrl: 'https://on.cypress.io/intercept',
+      },
+    },
+    Cookies: {
+      forbidden: {
+        message: `\`Cypress.Cookies.preserveOnce\` use is forbidden in ${cmd('switchToDomain')}. Consider using ${cmd('session')} instead.`,
+        docsUrl: 'https://on.cypress.io/session',
+      },
+    },
```

Comment:
"forbidden" reads a little dramatic to me. Maybe "has been deprecated its use is not supported in".

Also, let's make them all "Consider using ..." instead of "Considering migrating to using ..."

Reply:
fixed in 0b5f73c7839251b9a137b93a89c03ed5eff4655c